_install-and-configure/install-dashboards/plugins.md

```diff
@@ -144,14 +131,21 @@ This plugin adds a new Gantt chart visualization.
 
 {% endcomment %}
 
-## List installed plugins
+## Viewing a list of installed plugins
 
-To check your installed plugins:
+To view the list of installed plugins from the command line, use the following command:
 
 ```bash
 sudo bin/opensearch-dashboards-plugin list
 ```
 
+To view a list of plugins in the OpenSearch Dashboards UI:
+
+1. Connect to OpenSearch Dashboards.
+2. Select 
```

Comment:
I'm assuming there should be more text here.

Reply:
Deleted from this page, as it's covered in the Dashboard app section.